pkg/actions/nodegroup/update.go

```diff
@@ -0,0 +1,28 @@
+package nodegroup
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/kris-nova/logger"
+
+	"github.com/weaveworks/eksctl/pkg/managed"
+)
+
+func (m *Manager) Update() error {
+	ngName := m.cfg.ManagedNodeGroups[0].Name
+	_, err := m.ctl.Provider.EKS().DescribeNodegroup(&eks.DescribeNodegroupInput{
+		ClusterName:   &m.cfg.Metadata.Name,
+		NodegroupName: &ngName,
+	})
+
+	if err != nil {
+		if managed.IsNotFound(err) {
+			return fmt.Errorf("update is only supported for managed nodegroups; could not find one with name %q", ngName)
```

Comment:
slightly confusing because it could be either:

1. it doesn't exist
2. it exists as a `unmanaged` nodegroup

I think it might be worth distinguishing the two error cases, since this command might add a lot of confusion for users who try to update `unmanaged` nodegroups

Reply:
that's true, thank you, I'll find another way :)

Reply:
would it be less confusing and capture both error cases if the error message was simply reworded to `could not find managed nodegroup with name %q`?

Reply:
> would it be less confusing and capture both error cases if the error message was reworded to could not find managed nodegroup with name %q?

sounds good!